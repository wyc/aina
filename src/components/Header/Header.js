import React from 'react';

import { Link } from 'gatsby';

const Header = () => {
  return (
    <header className='Header'>
      <Link to='/'>Home</Link>
    </header>
  );
};

export default Header;
