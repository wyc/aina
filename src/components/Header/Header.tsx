import React, { ReactElement, ReactNode } from 'react';

import { Link } from 'gatsby';

export interface IHeaderProps {
  children?: ReactNode;
}

const Header = ({ children }: IHeaderProps): ReactElement => {
  return (
    <header className='Nav Row Header'>
      <Link to='/'>Home</Link>
      {children}
    </header>
  );
};

export default Header;
