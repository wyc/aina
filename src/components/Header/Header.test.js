import React from 'react';
import renderer from 'react-test-renderer';

import Header from './Header';

describe('Header', () => {
  it('renders correctly', () => {
    const props = { className: 'Header' };
    const component = renderer.create(<Header {...props} />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
