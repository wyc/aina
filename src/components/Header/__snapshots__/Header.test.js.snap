// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="Header"
>
  <a
    aria-current="page"
    className=""
    href="/"
    onClick={[Function]}
    onMouseEnter={[Function]}
    style={Object {}}
  >
    Home
  </a>
</header>
`;
